.travis.yml

```diff
@@ -17,6 +17,8 @@ before_install:
   - sudo echo "deb https://apt.repos.intel.com/openvino/2020 all main" | sudo tee - a /etc/apt/sources.list.d/intel-openvino-2020.list
   - sudo apt-get update
   - sudo apt-get install -y --no-install-recommends intel-openvino-dev-ubuntu16-2020.2.130
+  - curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py --user
+  - python3 -m pip install cloudpickle==1.3.0 future==0.18.2 gym==0.17.2 pyglet==1.5.0 scipy==1.5.0
```

Comment:
Please replace to the following at the beginning

```yml
- sudo apt-get install -y megatools python3-numpy python3-pip
- python3 -m pip install --upgrade numpy
- python3 -m pip install -r modules/8_reinforcement_learning/requirements.txt
```

Reply:
Applied, but I removed `- python3 -m pip install numpy` since `requirements.txt` has it.